core/ai-components/AIResponse/ChatBox.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { TBaseHtmlProps } from '../common.type';
+
+interface ChatBoxProps extends TBaseHtmlProps<HTMLDivElement> {
+  /**
+   * Pass children to `AI Response`
+   */
+  children: React.ReactNode;
+  /**
+   * Stores custom testing data private to the component.
+   */
+  'data-test'?: string;
+  /**
+   * Adds className to `AI Response`
+   */
+  className?: string;
+}
+
+export const ChatBox = (props: ChatBoxProps) => {
+  const { children, className, ...rest } = props;
+
+  const chatBoxClassNames = classNames(
+    {
+      'AIResponse-box': true,
+    },
+    className
+  );
+
+  return (
+    <div className={chatBoxClassNames} data-test="DesignSystem-AI-Response-Box" {...rest}>
```

Comment:
please make this `DesignSystem-AIResponse-Box` as we are using the component name as `AIResponse`, also make the same change if required in other file also.